client/src/components/PageLayout/Header.tsx

```diff
@@ -123,9 +130,9 @@ export const Header: React.FC = () => {
             )}
           </Box>
           {user && (
-            <NextLink passHref href="/profile">
+            <Link width="100%" href="/profile">
               <Avatar user={user} cursor="pointer" />
-            </NextLink>
+            </Link>
```

Comment:
I am worried how screen reader read this, but I don't have one and reading about it in text doesn't give a good concept of how they actually read it. 😞

Reply:
with windows narrator, it says "link the owner". but this isn't link to the owner it's link to profile, we need to change the context of the link to mention so, something like "link to your profile"